dokan/directory.c

```diff
@@ -23,9 +23,11 @@ with this program. If not, see <http://www.gnu.org/licenses/>.
 #include "fileinfo.h"
 #include "list.h"
 
+#ifdef _MSC_VER
 #if _MSC_VER < 1300 // VC6
```

Comment:
`#if defined(_MSC_VER) && _MSC_VER < 1300`

But the question is whether we really do support VC6 in 2020. If not, we might as well remove this. @Liryna What do you think?